olive/evaluator/olive_evaluator.py

```diff
@@ -183,8 +156,14 @@ def get_user_config(metric: Metric):
         eval_func = user_module.load_object(evaluate_func)
 
         if metric.user_config.input_names and metric.user_config.input_shapes and not dataloader and not eval_func:
-            dataloader = DummyDataloader(
-                metric.user_config.input_names, metric.user_config.input_shapes, metric.user_config.input_types
+            dataloader = (
```

Comment:
>( [](http://example.com/codeflow?start=25&length=1)

why do we need the outer side parenthesis?

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/936551ff-979a-4513-8ccf-ee276dd87318)

Actually, it was formatted by linter.